src/hub/action_request.ts

```diff
@@ -355,7 +355,15 @@ export class ActionRequest {
       } else {
         return sanitizeFilename(`looker_file_${Date.now()}.${this.attachment.fileExtension}`)
       }
+    } else if (this.formParams.format) {
+      if (this.scheduledPlan && this.scheduledPlan.title) {
+        return sanitizeFilename(`${this.scheduledPlan.title}.${formatToFileExtension(this.formParams.format)}`)
+      } else {
+        return sanitizeFilename(`looker_file_${Date.now()}.${formatToFileExtension(this.formParams.format)}`)
+      }
     }
+    winston.warn("Couldn't infer file extension from action request, using default filename scheme")
+    return sanitizeFilename(`looker_file_${Date.now()}`)
```

Comment:
I don't think AH support whitelabel (@phillipperalez, correct me if i'm wrong here). They can host AH themselves as well, and change the prefix. I'd say it's out of the scope of this since it's the same as L356: https://github.com/looker/actions/pull/343/files#diff-69063cbe99915fa7386cdacf954948c5L356